Neos.Flow/composer.json

```diff
@@ -38,6 +38,7 @@
         "doctrine/migrations": "^1.8",
         "doctrine/dbal": "^2.9",
         "doctrine/common": "^2.4",
+        "doctrine/inflector": "~1.3.0",
```

Comment:
I'd prefer the "dominant style" of `^1.3` here, TBH.

Reply:
Used this way because inflector 1.4 has the old API deprecated with a trigger warning, so that would lead to a lot of warning loggings suddenly. Since we should switch to the inflector 2.0+ API in Flow 6.0 any way (see comments above), this will only end up in the 5.3 branch.